src/phasorpy/phasor.py

```diff
@@ -2682,8 +2682,8 @@ def phasor_filter(
 ) -> tuple[NDArray[Any], NDArray[Any]]:
     """Return filtered phasor coordinates.
 
-    By default, a median filter is applied to the real and imaginary
-    components of phasor coordinates once with a kernel size of 3
+    By default, a median filter is applied independently to the real and
```

Comment:
Adding the 'independently' is a good idea, since I have been asked in some occasions if this is the case and it wasn't clear before in the description.